tests/test_crm.py

```diff
@@ -100,7 +100,11 @@ def test_CrmFdbEntry(dvs, testlog):
 
 def test_CrmIpv4Route(dvs, testlog):
 
-    dvs.runcmd("ifconfig Ethernet0 10.0.0.0/31 up")
+    config_db = swsscommon.DBConnector(swsscommon.CONFIG_DB, dvs.redis_sock, 0)
+    intf_tbl = swsscommon.Table(config_db, "INTERFACE")
+    fvs = swsscommon.FieldValuePairs([("NULL","NULL")])
+    intf_tbl.set("Ethernet0|10.0.0.0/31", fvs)
+    dvs.runcmd("ifconfig Ethernet0 up")
```

Comment:
Shouldn't use ifconfig. It's deprecated and removed and the source of lots of problems. We shouldn't have to rewrite all the unit tests if at some point we decide not to include net-tools pkg.